src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -260,6 +260,9 @@
   <data name="InvalidContinuationToken" xml:space="preserve">
     <value>The continuation token is invalid.</value>
   </data>
+  <data name="InvalidReIndexParameterValue" xml:space="preserve">
+    <value>Specified argument was out of the range of valid values. (Parameter '{0}'). Please specify different value within a range.</value>
```

Comment:
>range [](http://example.com/codeflow?start=127&length=5)

Would be nice if we actually provided interval here.

Reply:
Agree!